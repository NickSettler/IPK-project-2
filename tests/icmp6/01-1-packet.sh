./ipk-sniffer -i en0 --icmp6