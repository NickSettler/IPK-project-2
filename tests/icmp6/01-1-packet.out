timestamp: 2023-04-17T19:48:48.961+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 94:08:53:2e:50:cb
frame length: 70 bytes
src IP: 2a00:1028:83a2:25ea:cc68:af55:570a:888b
dst IP: fd7c:4710:a20f:4add:c478:8b37:af2c:5b15

0x0000: 94 08 53 2e 50 cb b0 be 83 4a c4 b0 86 dd 60 03  ..S.P....J....`.
0x0010: 08 00 00 10 3a 40 2a 00 10 28 83 a2 25 ea cc 68  ....:@*..(..%..h
0x0020: af 55 57 0a 88 8b fd 7c 47 10 a2 0f 4a dd c4 78  .UW....|G...J..x
0x0030: 8b 37 af 2c 5b 15 80 00 51 f9 cc 2f 00 02 64 3d  .7.,[...Q../..d=
0x0040: 86 80 00 0e ac 48                                .....H

